client/src/pages/Dashboard.js

```diff
@@ -0,0 +1,291 @@
+import React from "react";
+
+import {
+  Typography,
+  Container,
+  Box,
+  Avatar,
+  Button,
+  Card,
+  CardMedia,
+  CardContent,
+  Divider,
+  AppBar,
+  Toolbar,
+} from "@material-ui/core";
+
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+
+import projectpic1 from "../staticImages/projPicture1.png";
+import projectpic2 from "../staticImages/projPicture2.png";
+import projectpic3 from "../staticImages/projPicture3.png";
+import profpic1 from "../staticImages/profpic1.png";
+import logo from "../staticImages/ic-logo.png";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    color: "black",
+  },
+  appBar: {
+    background: "white",
+    color: "black",
+    elevation: 0,
+    "&MuiPaper": {
```

Comment:
hm this looks interesting, is this working? thought you might need a space inbetween like `& .MuiPaper` or something

Reply:
That is actually unnecessary, forgot to remove it